src/Symfony/Component/Mailer/Transport/Smtp/EsmtpTransport.php

```diff
@@ -132,12 +140,8 @@ private function doEhloCommand(): string
                 throw new TransportException('Unable to connect with STARTTLS.');
             }
 
-            try {
-                $response = $this->executeCommand(sprintf("EHLO %s\r\n", $this->getLocalDomain()), [250]);
-                $this->capabilities = $this->parseCapabilities($response);
-            } catch (TransportExceptionInterface) {
-                return parent::executeCommand(sprintf("HELO %s\r\n", $this->getLocalDomain()), [250]);
-            }
+            $response = $this->executeCommand(sprintf("EHLO %s\r\n", $this->getLocalDomain()), [250]);
```

Comment:
Why do we no longer need to catch transport exceptions anymore here?

Reply:
Here, we know that EHLO is supported, so there is no need to fall back to HELO.